packages/angular/src/generators/library/library.ts

```diff
@@ -58,7 +58,8 @@ export async function libraryGenerator(host: Tree, schema: Partial<Schema>) {
     name: options.name,
     prefix: options.prefix,
     entryFile: 'index',
-    skipPackageJson: !(options.publishable || options.buildable),
+    skipPackageJson:
+      options.skipPackageJson || !(options.publishable || options.buildable),
```

Comment:
Should these be kept for backward compability?

Reply:
Those are needed. We don't want to install anything when the library is not buildable or publishable.